trip-advisor-service/db/seed.js

```diff
@@ -103,9 +113,7 @@ class Attraction {
       suggestedDuration: randomGenerator(0, 200),
       address: randomGenerator(null, addresses),
     };
-    this.imageUrl = [urlGen(id),
-      urlGen(randomGenerator(0, 100)), urlGen(randomGenerator(0, 100)),
-      urlGen(randomGenerator(0, 100)), urlGen(randomGenerator(0, 100))];
+    this.imageUrl = [urlGen(id), ...picsListGen()];
```

Comment:
does doing it this way mean you'll have any number of pics assigned to a document between 2 and 30 instead of a set 5 for each?

Reply:
yes! @sabateklu